src/data/set/finite.lean

```diff
@@ -457,6 +461,12 @@ lemma finite_le_nat (n : ℕ) : finite {i | i ≤ n} := ⟨set.fintype_le_nat _
 
 lemma finite_lt_nat (n : ℕ) : finite {i | i < n} := ⟨set.fintype_lt_nat _⟩
 
+lemma exists_gt_nat_of_infinite {s : set ℕ} (hs : infinite s) (n : ℕ) : ∃ m, m ∈ s ∧ n < m :=
```

Comment:
There is a more mathliby formulation of this one, using filters, as `∃ᶠ n in at_top, n ∈ s`. And it would follow from the general version replacing `at_top` with `cofinite` (in any type), and the fact that, on naturals, `at_top = cofinite`.
Are the imports already available to state it like that? Otherwise, it might be worth adding the filter version somewhere (unless it is already available)?

Reply:
We already have [the filter version](https://leanprover-community.github.io/mathlib_docs/order/filter/cofinite.html#nat.frequently_at_top_iff_infinite). I suggest to rename:
```suggestion
lemma infinite.exists_nat_lt {s : set ℕ} (hs : infinite s) (n : ℕ) : ∃ m ∈ s, n < m :=
```